server/src/database.js

```diff
@@ -56,25 +52,26 @@ const createStore = () => {
       autoIncrement: true,
     },
     createdAt: Sequelize.DATE,
-    firstName: {type: Sequelize.STRING, allowNull: false},
-    lastName: {type: Sequelize.STRING, allowNull: false},
-    googleId: Sequelize.STRING,
-    facebookId: Sequelize.STRING,
-    profileImgUrl: Sequelize.STRING,
+    firstName: {type: Sequelize.TEXT, allowNull: false},
```

Comment:
Why are we switching to TEXT here and everywhere? Other than selfDescription, other fields should fit within 255 chars no?